content/privacy-policy.md

```diff
@@ -0,0 +1,139 @@
+---
+html: privacy-policy.html
+parent: xrp-ledger-overview.html
+blurb: 
+subtitle: Privacy Policy
+top_nav_grouping: About
+sidebar: disabled
```

Comment:
Just curious, why disable the sidebars? IMO the in-page navigation (on the right) is actually useful here. And the links on the left aren't bad either. Preview of how it looks with the default sidebars on:

![2023-01-19_114110_903177860](https://user-images.githubusercontent.com/7515597/213543845-b147b99a-417f-492e-91ee-8f8d94ca1ec0.png)

Reply:
Fixed